course/content.py

```diff
@@ -648,9 +648,7 @@ def look_up_git_object(repo: "dulwich.Repo",
         raise ObjectDoesNotExist(_("symlink nesting depth exceeded "
             "while locating '%s'") % full_name)
 
-    # https://github.com/inducer/relate/pull/556
-    # FIXME: https://github.com/inducer/relate/issues/767
-    name_parts = os.path.normpath(full_name).split(os.sep)
+    name_parts = list(os.path.split(os.path.normpath(full_name)))
```

Comment:
The test fails because `os.path.split` result in a 2-tuple, not split the path into path elements.
However, change to 
```
name_parts = os.path.normpath(full_name).split("/")
```
also fails in Windows.

@inducer